middleware/src/middleware.go

```diff
@@ -913,6 +908,43 @@ func (middleware *Middleware) SetLoginPassword(args rpcmessages.SetLoginPassword
 	}
 }
 
+// FinalizeSetupWizard finalizes the setup wizard by setting a redis key,
+// enabling bitcoin core and relates services and starting bitcoin code and
+// related services.
+func (middleware *Middleware) FinalizeSetupWizard() rpcmessages.ErrorResponse {
+	log.Println("Finalizing the setup wizard.")
+
+	err := middleware.redisClient.SetString(redis.BaseSetupDone, "1")
+	if err != nil {
+		log.Printf("Failed to finalize the setup wizard: %s", err)
+		return middleware.redisClient.ConvertErrorToErrorResponse(err)
+	}
+
+	out, err := middleware.runBBBConfigScript([]string{"enable", "bitcoin_services"})
+	if err != nil {
+		errorCode := handleBBBScriptErrorCode(out, err, []rpcmessages.ErrorCode{})
+
+		return rpcmessages.ErrorResponse{
+			Success: false,
+			Message: strings.Join(out, "\n"),
+			Code:    errorCode,
+		}
+	}
+
+	out, err = middleware.runBBBSystemctlScript([]string{"start-bitcoin-services"})
```

Comment:
I think documentation comments here would be really nice here to quickly explain the difference between
`middleware.runBBBConfigScript([]string{"enable", "bitcoin_services"})` and ` middleware.runBBBSystemctlScript([]string{"start-bitcoin-services"})`, why both are needed in this order for anyone looking at this in the future.